### ISSUE TITLE
Support for node self referencing.  This was closed in 4.5 beta but is not fixed in later versions => Self-referencing a package using its name #38675

### ISSUE THREAD

**Issue by FranzZemen** [microsoft/TypeScript#51633]
See the closed issue.  It definitely appears not fixed and I have the same issue as the those posting after #38675

**Comment by andrewbranch** [comment]
To expedite the triage process, we need everyone to follow the issue template and instructions.

When you clicked "Create New Issue", the issue form was pre-populated with a template and some instructions. We need you to read those instructions _completely_, follow them, and then fill in _all_ the fields in that template.

We are not able to assist with issues that don't follow the template instructions as they represent a significantly larger amount of work compared to issues which are correctly specified. Thank you for understanding.
